docs/README.md

```diff
@@ -24,10 +24,10 @@ The docs can be built with:
 make docs-latest
 ```
 
-To serve the docs and watch for changes, run:
+To makes and serve the docs and watch for changes, run:
 
 ```shell
-make serve-latest
+make serve-docs-latest
```

Comment:
This relates to the Makefile in parent dir. There is another Makefile under /docs which this is referring to.

But should probably add line to state that `cd /docs` and still add command for parent Makefile.

Reply:
Added a help to the docs makefile (call it from normal makefile)